ui/components/app/multilayer-fee-message/multi-layer-fee-message.js

```diff
@@ -75,15 +78,15 @@ export default function MultilayerFeeMessage({
   return (
     <div className="multi-layer-fee-message">
       <TransactionDetailItem
-        key="total-item"
+        key="total-item-gas-fee"
```

Comment:
fixing the warning in console
<img width="987" alt="Screenshot 2023-04-06 at 00 37 06" src="https://user-images.githubusercontent.com/12678455/230236401-d3857a24-1cbe-4c46-a2cc-8f050f4ee5b1.png">